canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -150,6 +180,7 @@ export default class PropertiesController {
 		this.reactIntl = intl;
 		// set initial property values
 		if (this.form) {
+			this.oldControls = this.controls;
```

Comment:
instead of old maybe `prevControls` or maybe even just the controlIds if that's all we care about.